manifest.go

```diff
@@ -91,12 +84,22 @@ const (
 	pruneOptionNonGo          = "non-go"
 )
 
+// Constants to represents per-project prune uint8 values.
+const (
+	pvnone  uint8 = 0 // No per-project prune value was set in Gopkg.toml.
```

Comment:
Did you consider using `iota`?

Reply:
yes, i preferred this form because:

1. it is visually explicit, and the sole purpose of having these types is for the reader
2. while it happens to be true that iota enumerates the first three values traditionally used in a trinary, iotas are not for expressing trinaries.